src/ldc/eh/common.d

```diff
@@ -441,6 +466,35 @@ extern(C) auto eh_personality_common(NativeContext)(ref NativeContext nativeCont
     // landing pad (will be zero if there are none).
     size_t actionTableStartOffset;
 
+    version (SjLj_Exceptions)
+    {
+        if (ip < 0)
+            return nativeContext.continueUnwind();
+
+        if (ip == 0)
+        {
+            // If ip is not present in the table, call terminate.  This is for
+            // a destructor inside a cleanup, or a library routine the compiler
```

Comment:
Its been so long since I wrote that that, it was probably based on what I saw in other implementations.  I'll just leave the first sentence "If ip ... call terminate."

I just noticed that libgcc just does a continue unwind when ip <= 0 also. https://github.com/gcc-mirror/gcc/blob/master/libgcc/unwind-c.c#L170.  Wait, that's confusing based on their comment.  Is that better behavior?